doc/sphinxext/math_symbol_table.py

```diff
@@ -140,13 +139,6 @@ def get_n(n, l):
     return []
 
 
-@cbook.deprecated("3.2", alternative="MathSymbolTableDirective")
-def math_symbol_table_directive(
```

Comment:
Should this get an entry in the removal changelog?

Reply:
No, I don't even know why it has a deprecation. This file is only used when building docs and isn't installed. The Matplolib sphinx extension is in `lib/matplotlib/sphinxext`.